src/soter/openssl/soter_ec_key.c

```diff
@@ -258,9 +261,14 @@ soter_status_t soter_engine_specific_to_ec_pub_key(const soter_engine_specific_e
     }
 #else
     if (compressed) {
-        param_name = OSSL_PKEY_PARAM_PUB_KEY; // "pub"
+        param_name = OSSL_PKEY_PARAM_PUB_KEY /* "pub" */;
```

Comment:
what the reason to change type of comments?

Reply:
Just made it look like in other places where it's a function argument

Reply:
Personally, I don't see much value in these comments in the first place. One should either hardcode strings with no further explanation, or use googlable constants, but not both. It's okay to comment why a particular constant is used if it is not obvious from context, but in many places it is obvious and does not require commentary, IMO.